test/util.sh

```diff
@@ -95,7 +95,7 @@ assert_response() {
 it_runs_free_enterprise() {
   info=$(curl $KONG_ADMIN_URI)
   msg_test "it does not have ee-only plugins"
-  [ "$(echo $info | jq -r .plugins.available_on_server.collector)" != "true" ]
+  [ "$(echo $info | jq -r .plugins.available_on_server.canary)" != "true" ]
```

Comment:
I'm guessing canary is a newer EE plugin that didn't have a similar test case before?

Reply:
I literally just replaced the old ee plugin by another one.